### DISCUSSION TITLE
Release v1.14.9 / 5.69.9

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#4254] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### Release Notes

This build fixes various issues and adds a few features, for a full list of changes please review the [change log](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1149--5699---2024-09-19).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.14.9'>Release v1.14.9 / 5.69.9</a>.</em>

**Comment by bastik-1001** [comment]
The file: `Sandboxie-Plus-x64-v1.14.9.exe` is signed and has the following hashes:

```
CRC32: 9D39AE7B
CRC64: 6449415B0AADFBA7
MD5: EFFCEFAD2C3281A17E4485617821CE20
SHA-1: 331E4DA82275C868D7E5BBC9446217B0F0ACC337
SHA-256: BC1060F8FD890B614DE4AF48C401AC25BB5774EBFE74F17B66AE07AB57F78B2E
SHA-512: C0932047C2107E1B5518E328E35ACDAFC778736D813C2E1361616B7393142A4FCD40818901DC6BF1A1EBC0DEB7D380A9B9D8EC560A72FFD4CABC938DBF5377FB
```
